validations/src/tests.rs

```diff
@@ -145,6 +145,50 @@ fn mint_invalid_epoch() {
     );
 }
 
+#[test]
+fn mint_invalid_pkh() {
+    let epoch = 0;
+    let reward = block_reward(epoch);
+    let total_fees = 100;
+    let output1 = ValueTransferOutput {
+        pkh: Default::default(),
+        value: total_fees,
+        time_lock: 0,
+    };
+    let output2 = ValueTransferOutput {
+        pkh: PublicKeyHash::from_bytes(&[1; 20]).unwrap(),
+        value: reward,
+        time_lock: 0,
+    };
+    let mint_tx = MintTransaction::new(epoch, vec![output1, output2]);
+    let x = validate_mint_transaction(&mint_tx, total_fees, epoch);
+    // Error: different pkhs in mint transaction
+    assert_eq!(
+        x.unwrap_err().downcast::<BlockError>().unwrap(),
+        BlockError::DifferentPkhsInMint
+    );
+}
+
+#[test]
+fn mint_split_valid() {
+    let epoch = 0;
+    let reward = block_reward(epoch);
+    let total_fees = 100;
+    let output1 = ValueTransferOutput {
+        pkh: Default::default(),
+        value: total_fees,
```

Comment:
Add another test where one of the values is 0. Also, we should put a strict limit on the size of the `MintTransaction` to prevent miners from filling blocks with junk.